sections/sentinels.tex

```diff
@@ -0,0 +1,183 @@
+    \item Killed enemy units do not respawn. 
+\end{itemize}
+
+\subsection*{\MakeUppercase{Boss army}}
+
+\begin{itemize}
+    \item On Round 10 a Boss Army spawns and is different from previous armies, because it gets reinforced 
+    from a fixed pool of neutral units (reinforcement pool = number in brackets in the table below).
+    \item At the beginning of the 2$^{nd}$ combat round and every following round 2 units reinforce the
+    remaining army from the shuffled reinforcement pool (up to the maximum of 5 units) to
+    continue the fight until either the player retreats or every neutral unit from the reinforcement
+    pool is killed. \textit{If the number of enemy units at the beginning of a round is 1 or lower, draw up to 
+    a total of 4 units (e.i. there are always 4 or 5 enemy units on the board after reinforcing, if available).}
+    \item Place the reinforcing units on the enemy base line starting from the left with the lowest initiative
+    (\svg{unit_ranged} units first). If necessary continue on the next line (in the same order).
+    \item When a player retreats the enemy army draws up to the "minimum" of 4 units and takes these
+    as starting units to the next fight. 
+\end{itemize}
+
+\hommtable[]{18}{
+  \centering
+  \medskip
+  \textbf{Strength of Enemy Armies}\\
+  \bigskip
+
+  \newcommand{\bronze}[0]{\svg[12]{bronze}}
+  \newcommand{\silver}[0]{\svg[12]{silver}}
+  \newcommand{\golden}[0]{\svg[12]{golden}}
+  \newcommand{\azure}[0]{\svg[12]{azure}}
+
+  \begin{tabularx}{\linewidth}{p{0.15\linewidth}XXXX} & \darkcell{Rounds 6 + 7} & \darkcell{Rounds 8 + 9} & \darkcell{Round 10}\\
+    \darkcell[1.4]{Easy} 
+        & \lightcell[1.4]{\bronze \silver \silver \silver \golden} 
+        & \lightcell[1.4]{\silver \silver \silver \golden \golden} 
+        & \lightcell[1.4]{\silver \silver \golden \azure \footref{azure} \linebreak 
+            (2\bronze, 4\silver, 3\golden, 1\azure)}\\
+    \darkcell[1.4]{Normal} 
+        & \lightcell[1.4]{\silver \silver \silver \golden \golden} 
+        & \lightcell[1.4]{\silver \silver \silver \golden \azure} 
+        & \lightcell[1.4]{\golden \golden \golden \azure \footref{azure} \linebreak 
+            (2\silver, 7\golden, 1\azure)}\\
+    \darkcell[1.4]{Hard} 
+        & \lightcell[1.4]{\silver \silver \golden \golden \golden} 
+        & \lightcell[1.4]{\silver \silver \golden \golden \azure}
+        & \lightcell[1.4]{\golden \golden \azure \azure \footref{azure} \linebreak 
+            (2\silver, 6\golden, 2\azure)}\\
+    \darkcell[1.4]{Impossible} 
+        & \lightcell[1.4]{\silver \golden \golden \golden \golden} 
+        & \lightcell[1.4]{\golden \golden \golden \golden \azure} 
+        & \lightcell[1.4]{\golden \azure \azure \azure \footref{azure} \linebreak 
+            (7\golden, 3\azure)}\\
+  \end{tabularx}
+}
+
+\subsection*{\MakeUppercase{Credits}}
+
+\textbf{Author:} silence70011
```

Comment:
I think the author should be at the top.

Reply:
done